VideoLocker/src/main/java/org/edx/mobile/module/db/impl/DbOperationBase.java

```diff
@@ -38,7 +38,7 @@ public T requestExecute(SQLiteDatabase db) {
             }
         }
         
-        return null;
+        return getDefaultValue();
```

Comment:
this change may break existing logic.  
because null is different from default value. 

we should valid the data model before insert it into the database table.